src/db.ts

```diff
@@ -918,33 +933,39 @@ export abstract class Database {
           continue;
         }
 
-        updateJobs.recorded(values[1], values[0], values[4]);
```

Comment:
Hm, is there a reason to move the position of this?

Reply:
no there was not :)